.gitattributes

```diff
@@ -0,0 +1,3 @@
+tests/snapshots/*.snap linguist-generated=true
+tests/snapshots/ linguist-generated=true
```

Comment:
Needs .gitattributes at the top level? As we have a monorepo in case this is required, can you prepend `"packages/libs/error-stack"` to make them "absolute"?

Reply:
I thought it was, but turns out I am wrong (it was once in the error-stack repo, but then moved it up). According to the official documentation:

> When deciding what attributes are assigned to a path, Git consults $GIT_DIR/info/attributes file (which has the highest precedence), .gitattributes file in the same directory as the path in question, and its parent directories up to the toplevel of the work tree (the further the directory that contains .gitattributes is from the path in question, the lower its precedence). Finally global and system-wide files are considered (they have the lowest precedence).

(https://www.git-scm.com/docs/gitattributes) it seems like I can move it back into the error-stack repo.